docs/self-managed/zeebe-deployment/kubernetes/helm/installing-helm.md

```diff
@@ -1,45 +1,47 @@
 ---
```

Comment:
Is there a reason we keep the installation guide in Zeebe deployment? I was wondering if it makes sense to have something for whole platform?

Reply:
@Hafflgav

Reply:
I think this is related to my comment here https://github.com/camunda-cloud/camunda-cloud-documentation/issues/609#issuecomment-1041467773

Reply:
Jup I have this in mind. As of now I wanted to get past the content before further changing the structure of the document. :)

Reply:
Great! Sorry I missed this comment from Zell, thanks!